content/docs/quickstart/postgres.md

```diff
@@ -2,10 +2,16 @@
 title: Run a Postgres App
 ---
 
-Since Neon compute is regular Postgres any Postgres app will run out of the box.
-You can also use all the standard client utilities like ‘psql’ and ‘pg_dump’, and client libraries and drivers to connect.
+Since Neon compute is regular Postgres, any Postgres app will run out of the box.
+You can also use all the standard client utilities like `psql` and `pg_dump`, client libraries and drivers to connect.
 
-All connection to Postgres are done via an endpoint hosted at `pg.neon.tech`. You can try it right away by running `psql -h pg.neon.tech` in your terminal.
+All connection to Postgres are done via an endpoint hosted at `pg.neon.tech`.
```

Comment:
probably `is done`?

Reply:
@duskpoet "is done" is the correct way. 

It should be **All connections to Postgres are done via an endpoint hosted at `pg.neon.tech`**.

Unless you can only make one, singular connection to Postgres, please correct me if I am wrong.

Reply:
@duskpoet 

Please refer to this commit, https://github.com/neondatabase/website/pull/59/commits/18c9b66d62c94fa00939aad97366235a686133b2